kfs-core/src/main/java/edu/arizona/kfs/sys/batch/service/impl/BuildingImportServiceImpl.java

```diff
@@ -37,74 +36,72 @@
 		Building building = null;
 		Building buildingCodeMatch;
 		String routeCodeValue;
 		String campusCode;
 		String reportMsg;
 		PrintWriter outReportErrorWriter = null;
-		PrintWriter outReportWriter = null; 
+		PrintWriter outReportWriter = null;
 		BuildingExtension buildingExt;
 		CampusBo campus;
 		Map<String, String> routecodeToCampuscodeMap = buildingAndRoomImportDao.PopulateRoutecodeToCampusCodeMap();
-		
-		try{
+
+		try {
 			outReportWriter = setupReportOutputFiles(outReportWriter);
 			outReportErrorWriter = setupErrorReportOutputFiles(outReportErrorWriter);
-			writetoReportOutputFilesHeader(outReportWriter, outReportErrorWriter);
+			writetoReportOutputFilesHeader(outReportWriter,outReportErrorWriter);
 			Collection<ArchibusBuildings> archibusBuildings = businessObjectService.findAll(ArchibusBuildings.class);
-						
-			// This will look for all records in Archibus building then do an update or insert into buildings
+
+			// This will look for all records in Archibus building then do an
+			// update or insert into buildings
 			for (ArchibusBuildings archBuilding : archibusBuildings) {
+				LOG.info("Processing Record: " + archBuilding.toString());
 				reportMsg = "";
-				
-				// Look for a valid routecode in kfs
 				routeCodeValue = archBuilding.getRouteCode();
-				
+
 				RouteCode routecode = (RouteCode) businessObjectService.findBySinglePrimaryKey(RouteCode.class, routeCodeValue);
-				
+
 				// report error if routecode is not set up in KFS
 				if (!KFSParameterKeyConstants.HYPHEN.equalsIgnoreCase(routeCodeValue) && ObjectUtils.isNull(routecode)) {
-					LOG.debug("Routecode value of " + routeCodeValue + " is not set up in KFS.");
 					writetoErrorReportOutputFile(outReportErrorWriter, reportMsg, archBuilding.getCampusCode(), archBuilding.getBuildingCode(), "Routecode value of " + routeCodeValue + " is not set up in KFS.");
 					continue;
 				}
 				// if building is active then continue trough
 				if (KFSParameterKeyConstants.STRING_A.equalsIgnoreCase(archBuilding.getActive())) {
-					if(StringUtils.isBlank(routecodeToCampuscodeMap.get(routeCodeValue))) {
+					if (StringUtils.isBlank(routecodeToCampuscodeMap.get(routeCodeValue))) {
 						campusCode = KFSParameterKeyConstants.MAIN_CAMPUSCODE;
-					}
+					} 
 					else {
 						campusCode = routecodeToCampuscodeMap.get(routeCodeValue);
-						
+
 						// Look for a valid Campus Code in KFS
 						Map<String, String> pkeys = new HashMap<String, String>();
-                        pkeys.put("code", campusCode);
-                        pkeys.put("active", "Y");
-                        campus = (CampusBo) businessObjectService.findByPrimaryKey(CampusBo.class, pkeys);
-						
+						pkeys.put("code", campusCode);
+						pkeys.put("active", "Y");
+						campus = (CampusBo) businessObjectService.findByPrimaryKey(CampusBo.class, pkeys);
+
 						if (ObjectUtils.isNull(campus)) {
-							LOG.debug("Campus Code is not valid, record not saved to building");
```

Comment:
We may want to leave in or add some debugging or info messages within this whole method prepareBuildingImport in case there are errors during runtime.